server/src/main/java/io/druid/server/router/PriorityTieredBrokerSelectorStrategy.java

```diff
@@ -50,18 +49,10 @@ public PriorityTieredBrokerSelectorStrategy(
 
     if (priority < minPriority) {
```

Comment:
Two branches of this if block are the same now.

Reply:
combined the two branches into one